### DISCUSSION TITLE
Export to Linux

### DISCUSSION THREAD

**Discussion by ghost** [nem0/LumixEngine#1515] [Ideas]
Hey, great work on the engine. I plan to learn the Lumix Engine within the next couple of weeks. I'm a Linux user, and I see that there is Linux support. Will I be able to export a finished game to Linux as well? I read through the docs and the itch page, but I couldn't find anything about that.

**Comment by nem0** [comment]
Hi, currently Lumix does not run on Linux. The DX11 and OpenGL renderers have been removed, leaving only the DX12 renderer. I plan to make it work on Linux again, but not in the near future.

**Reply by ghost** [reply]
Thanks for the reply. I don't have a Windows PC that I can use for GameDev, so I'll have to wait to try it.

**Reply by raojm** [reply]
can replace renders with bgfx or  DiligentEngine  to support all platform?

**Comment by nem0** [comment]
You might have to wait a long time. The Linux version has a very low priority.
